### ISSUE TITLE
Secret_key_base is leaking through the email

### ISSUE THREAD

**Issue by thebravoman** [smartinez87/exception_notification#507]
### Steps to reproduce
1. Install 
2. Simulate an error
3. Receive an email
The email contains action_dispatch.secret_key_base

This is the secret_key_base that is leaking through the email. 

### Expected behavior
Hide the secret_key_base

### Actual behavior
It is in the email as action_dispatch.secret_key_base
### System configuration
**Rails version**:
6.0
**Ruby version**:
2.6.5
**Other configurations**:

**Comment by thebravoman** [comment]
I was wondering if you've had the chance to look at this. It seems to be a security issue as the secret key base is how most of the security on rails is handled. So I think it should not be exposed

**Comment by frenkel** [comment]
For me it shows up as:
```
     * action_dispatch.secret_key_base                         : [FILTERED]
```

Did you setup your `Rails.application.config.filter_parameters` correctly?

**Comment by thebravoman** [comment]
No, I did not realize that exception_notification was using filter_parameters

Should it be like 

```
Rails.application.config.filter_parameters += [:password, :secret_key_base]
```

Would you accept a PR that gives more information about filter_parameters?

**Comment by frenkel** [comment]
The default for a Rails 6 app is this, it is what I use:

config/initializers/filter_parameter_logging.rb :
```
# Be sure to restart your server when you modify this file.

# Configure sensitive parameters which will be filtered from the log file.
Rails.application.config.filter_parameters += [
  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
]
```

I'm not the original author, so I cannot accept your PR if you create one.

**Comment by thebravoman** [comment]
Got it.

I am running this platform since rails 2 and there was only one filtered
param.

Will add the rest.
